README.md

```diff
@@ -158,9 +158,9 @@ to decide whether to apply it.
 
 To gain a better understanding of how Elastic CI Stack works and how to use it most effectively and securely, see the following resources:
 
-* [Elastic CI Stack for AWS overview](https://buildkite.com/docs/agent/v3/elastic_ci_aws)
-* [Elastic CI Stack for AWS tutorial](https://buildkite.com/docs/tutorials/elastic-ci-stack-aws)
-* [Running Buildkite Agent on AWS](https://buildkite.com/docs/agent/v3/aws)
+* [Elastic CI Stack for AWS overview](https://buildkite.com/docs/agent/v3/elastic-ci-aws/elastic-ci-stack-overview)
+* [Installation and setup recommendations](https://buildkite.com/docs/agent/v3/aws)
+* [Elastic CI Stack for AWS Linux and Windows setup](https://buildkite.com/docs/agent/v3/elastic-ci-aws)
```

Comment:
```suggestion
* [Elastic CI Stack for AWS overview](https://buildkite.com/docs/agent/v3/elastic-ci-aws/elastic-ci-stack-overview)
* [Elastic CI Stack for AWS Linux and Windows setup](https://buildkite.com/docs/agent/v3/elastic-ci-aws)
```

Reply:
The link to "Installation and setup recommendations" (which I now clearly see needs further work on my part) is out of scope for Elastic CI Stack so doesn't need to be in the Readme.

Reply:
Done and committed :-)